### ISSUE TITLE
$('#form').serializeArray() gives empty value (Safari iOS)

### ISSUE THREAD

**Issue by SGE00** [RobinHerbots/Inputmask#1246]
Hi,

I have a form element with multiple fields, some of them with simple inputmask, like data-inputmask="'mask':'99:99'".

On Safari (iOS) following code:

var formdata = $('#myform').serializeArray();    

gives an empty value for any fields with an inputmask.

Edit: Even trying to access the value using: alert($("#a_field").val()) returns an empty value.

On a side note: I could not see in your documentation on which browsers/OS this jquery plugin should successfully work

Is there a list available ?

**Comment by Skulli** [comment]
Yeah just noticed this as well. When doing an ajaxSubmit via jquery.form, nothing is submited in the masked input fields.
See #1244 my solution there was wrong since i tested the solution in a non ajax form.

**Comment by SGE00** [comment]
Hi, thx for the feedback

$(".datepicker-input").inputmask 'dd.mm.yyyy',
  autoUnmask: true

that works, but I need the masked value but

$(".datepicker-input").inputmask 'dd.mm.yyyy',
  autoUnmask: false

does not seem to work (still empty value).

**Comment by RobinHerbots** [comment]
@SGE00 , @Skulli , 

I tested it on browserstack and the serializeArray works fine.  Can you create a jsfiddle with an example?

**Comment by Skulli** [comment]
For now i was not able to reproduce it on jsfiddle. It only happens on some fields for me. In my case a date field, where i also use some datepicker plugin. It also only happens when i submit the form via ajax. A regular POST works just fine. I will investigate in the next few days again.

**Comment by RobinHerbots** [comment]
@Skulli ,

Is this still an issue?

**Comment by Skulli** [comment]
Cant say anyhting about it, since i changed the job since then and not using this tool anymore.

**Comment by RobinHerbots** [comment]
@SGE00 ,

Is this still an issue?
